kratos/python_scripts/assign_vector_variable_process.py

```diff
@@ -108,3 +108,22 @@ def ExecuteFinalizeSolutionStep(self):
         """
         for process in self.aux_processes:
             process.ExecuteFinalizeSolutionStep()
+
+    @staticmethod
+    def GetSpecifications(settings):
+        # Set base empty specifications for processes
+        specifications = KratosMultiphysics.Parameters("""{
+            "required_solution_step_data_variables" : [],
+            "required_dofs" : [],
+            "nodal_flags_used" : [],
+            "elemental_flags_used" : [],
+            "condition_flags_used" : [],
+            "documentation" : "This process assigns a given value (vector) to the nodes belonging a certain submodelpart."
+        }""")
+
+        # Customize specifications according to input settings
+        specifications["required_solution_step_data_variables"].SetStringArray([settings["variable_name"].GetString()])
+        if any(settings["constrained"].values()):
+            specifications["required_dofs"].SetStringArray([settings["variable_name"].GetString()])
```

Comment:
We shall check if the fixity is set to add the dof. Isn't it?

Reply:
I think I'm not getting your point at all... Isn't this what we're actually doing with the `any()`?